### PR TITLE
[JENKINS-65398] terminology update

### PR DESCRIPTION
**Pull request by aHenryJard**
Removing offending terminology such as master,slave or whitelist/blacklist as part of [JENKINS-65398](https://issues.jenkins.io/browse/JENKINS-65398)

I tried to update whitelist/ black list with the context, feel free to tell me if it's not accurate.

- [x] Make sure you are opening from a **topic/feature/bugfix branch** (right side) and not your master branch!
- [x] Ensure that the pull request title represents the desired changelog entry
- [x] Please describe what you did
- [x] Link to relevant issues in GitHub or Jira
- [x] Link to relevant pull requests, esp. upstream and downstream changes
- [ ] Ensure you have provided tests - that demonstrates feature works or fixes the issue

### CODE REVIEW

File: src/main/java/com/cloudbees/plugins/credentials/CredentialsProviderFilter.java [hunk 1]
```diff
@@ -177,7 +177,7 @@ public String getDisplayName() {
         /**
          * Our constructor.
          *
-         * @param classNames the whitelist of class names.
+         * @param classNames the list of included class names.
```
Comment (jvz) [inline src/main/java/com/cloudbees/plugins/credentials/CredentialsProviderFilter.java:180]:
```suggestion
         * @param classNames the list of allowed class names.
```

To keep consistent with the other changes below.

File: src/main/resources/com/cloudbees/plugins/credentials/BaseCredentials/help-scope.html [hunk 2]
```diff
@@ -47,7 +47,7 @@
 
     <p>
     In general, a credential is defined in one place (e.g., the credentials configuration page under "Manage Jenkins")
-    and then used in another place (e.g., when connecting to a new SSH slave).
+    and then used in another place (e.g., when connecting to a new SSH agent).
```
Comment (jvz) [inline src/main/resources/com/cloudbees/plugins/credentials/BaseCredentials/help-scope.html:50]:
```suggestion
    and then used in another place (e.g., when connecting to a new SSH build agent).
```

File: src/main/resources/com/cloudbees/plugins/credentials/impl/CertificateCredentialsImpl/FileOnMasterKeyStoreSource/help-keyStoreFile.html [hunk 3]
```diff
@@ -23,5 +23,5 @@
  -->
 
 <div>
-    The path to the file on the Jenkins master node.
+    The path to the file on the Jenkins controller node.
```
Comment (jvz) [inline src/main/resources/com/cloudbees/plugins/credentials/impl/CertificateCredentialsImpl/FileOnMasterKeyStoreSource/help-keyStoreFile.html:26]:
```suggestion
    The path to the file on the Jenkins controller.
```

**Review by jvz — APPROVED** [review]

File: docs/user.adoc [hunk 4]
```diff
@@ -52,7 +52,7 @@ While the Credentials API plugin provides a default _internal_ credentials store
 If you are using the _internal_ credentials store to store *high value* credentials then you will need to lock down your Jenkins configuration:
 
 * Apply all of the recommendations from the  https://wiki.jenkins.io/display/JENKINS/Securing+Jenkins[Securing Jenkins] wiki page.
-* No builds on the master (ideally zero executors)
+* No builds on the controller (ideally zero executors)
```
Comment (daniel-beck) [inline docs/user.adoc:55]: Potentially wrong term

Comment (jvz) [reply docs/user.adoc:55]: What would you suggest?

Comment (daniel-beck) [reply docs/user.adoc:55]: Still undefined due to lack of followup for the last 8 months.

**Pull request merged**